### PR TITLE
adaptive Main page

### LINKED ISSUE #154: Make adaptive Main page
1. Read guideline [here](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5)
2. Make adaptive design for `phones` and `tablet` for main page
3. This task only for project contributors not for new comers

### PR DESCRIPTION
**Pull request by od-hunter**
Resolves #154 
please review

### CODE REVIEW

**Comment by djeck1432** [comment]
@od-hunter Please resolve merge conflict

File: frontend/src/pages/spotnet/about/About.jsx [hunk 1]
```diff
@@ -34,29 +34,43 @@ const CardData = [
...
+        <div className="about-container">
+            {starData.map((star, index) => (
+                <Star key={index} style={{
+                    position: 'absolute',
+                    top: `${star.top}%`,
+                    left: `${star.left}%`,
+                    width: `${star.size}%`,
+                    height: `${star.size}%`
+                }}/>
+            ))}
```
Comment (whateverfw) [inline frontend/src/pages/spotnet/about/About.jsx:52]: Please use already existing `StarMaker` component instead

File: frontend/src/pages/spotnet/about/About.jsx
Comment (whateverfw) [inline frontend/src/pages/spotnet/about/About.jsx:1]: this file cause build errors

could you please fix it and verify by doing `npm run build` locally?

src/pages/spotnet/about/About.jsx
16.40 Syntax error: Unexpected token, expected "," (75:8) (75:8)

File: frontend/src/components/Footer/footer.css [hunk 2]
```diff
@@ -1,8 +1,36 @@
...
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style-type: none;
+}
+
```
Comment (whateverfw) [inline frontend/src/components/Footer/footer.css:28]: this seems to partially repeat functionality that already exists in `globals.css`, please combine missing parts into `globals.css` and remove this from `footer.css` and verify everything still works correctly

File: frontend/src/pages/spotnet/home/home.css [hunk 3]
```diff
@@ -1,3 +1,17 @@
...
+  --secondary-color: #1a1c24;
+  --blue-color: #74d6fd;
+  --primary-color: #0b0c10;
+  --black: #000;
+  --primary: #fff;
+  --secondary: #e7ecf0;
+  --button-gradient: linear-gradient(55deg, #74d6fd 0%, #e01dee 100%);
+  --button-gradient-hover: linear-gradient(55deg, #74d6fd 0%, #74d6fd 100%);
+  --button-gradient-active: linear-gradient(55deg, #58c4ef 0%, #58c4ef 100%);
+}
```
Comment (whateverfw) [inline frontend/src/pages/spotnet/home/home.css:14]: this seems to partially repeat functionality that already exists in `globals.css`, please combine missing parts into `globals.css` and remove this from `home.css` and verify everything still works correctly

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Review by whateverfw — CHANGES_REQUESTED** [review]
Also please resolve merge conflicts with `main` using footer and header versions from there and verifying it works well along with your other changes

Thanks

**Comment by whateverfw** [comment]
@od-hunter

File: frontend/src/components/header/Header.jsx
Comment (whateverfw) [inline frontend/src/components/header/Header.jsx:1]: this wasn't merged properly, just keep Header component that is on main (also css)

File: frontend/src/components/Footer/Footer.jsx
Comment (whateverfw) [inline frontend/src/components/Footer/Footer.jsx:1]: this wasn't merged properly, just keep Header component that is on main (also css)

**Review by whateverfw — CHANGES_REQUESTED** [review]
also fix build issues (check CI)

before you next commit please run `npm run build` and verify that nothing fails, thx

**Review by whateverfw — APPROVED** [review]

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**